src/components/Footer/ListLinks.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Link from './Link';
+
+const ListLinks = ({classNameContainer, classNameChild, children, links}) => (
+  <div className={classNameContainer}>
+    {children}
+    {links.map(link => (
+      <Link key={link.text || link.image} className={classNameChild}>
```

Comment:
Key must be unique. React reconciliation will fail if keys aren't unique.

Reply:
fixed